src/app/sliders/horizontal-slider/horizontal-slider.component.css

```diff
@@ -66,14 +66,10 @@ span {
     transform: translate(-50%, 0);
 }
 
-.bubble-span-left {
-    background: #344a69;
-    left: 0px;
-}
-
+.bubble-span-left,
 .bubble-span-right {
     background: #344a69;
-    right: 0px;
```

Comment:
We cannot remove right
![image](https://github.com/pib-rocks/cerebra/assets/144774775/a1c1c261-a97e-40aa-86ff-17b4f15ebbd6)